src/@dvcorg/gatsby-theme-iterative/components/Page/DefaultSEO/index.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import Helmet from 'react-helmet'
+
+import { MetaProps } from '@dvcorg/gatsby-theme-iterative/src/components/SEO'
+import getSiteMeta from '@dvcorg/gatsby-theme-iterative/src/queries/siteMeta'
+
+interface IDefaultSEOProps {
+  pathname: string
+}
+
+const metaImage = {
+  src: '/social-share.png',
+  width: '1200',
+  height: '630'
+}
+
+const DefaultSEO: React.FC<IDefaultSEOProps> = ({ pathname }) => {
+  const siteMeta = getSiteMeta()
+  const siteUrl = siteMeta.siteUrl
+  const metaTitle = siteMeta.title
+  const metaDescription = siteMeta.description
+  const metaKeywords = siteMeta.keywords
+  const fullUrl = siteUrl + pathname
+
+  const meta: MetaProps[] = [
+    {
+      name: 'description',
+      content: metaDescription
+    },
+    {
+      name: 'keywords',
+      content: metaKeywords
+    },
+    {
+      property: 'og:url',
+      content: fullUrl
+    },
+    {
+      property: 'og:site_name',
+      content: metaTitle
+    },
+    {
+      property: 'og:title',
+      content: metaTitle
+    },
+    {
+      property: 'og:description',
+      content: metaDescription
+    },
+    {
+      property: 'og:type',
+      content: 'website'
+    },
+    {
+      property: 'og:image',
+      content: siteUrl + metaImage.src
+    },
+    {
+      property: 'og:image:width',
+      content: metaImage.width
+    },
+    {
+      property: 'og:image:height',
+      content: metaImage.height
+    },
+    {
+      property: 'og:image:type',
+      content: 'image/png'
+    },
+    {
+      property: 'og:locale',
+      content: 'en_US'
+    },
+    {
+      name: 'twitter:site',
+      content: '@DVCorg'
+    },
+    {
+      name: 'twitter:creator',
+      content: '@DVCorg'
+    },
+    {
+      name: 'twitter:card',
+      content: 'summary'
+    },
+    {
+      name: 'twitter:title',
+      content: metaTitle
+    },
+    {
+      name: 'twitter:description',
+      content: metaDescription
+    },
+    {
+      name: 'twitter:image',
+      content: encodeURI(`${siteUrl}${metaImage.src}`)
+    },
+    {
+      name: 'twitter:image:alt',
+      content: 'The DVC logo on a gray background'
+    }
+  ]
+
+  console.log('Wumbo!')
+
+  return (
+    <Helmet
+      htmlAttributes={{
+        lang: 'en'
+      }}
+      defaultTitle={metaTitle}
+      meta={meta}
+      link={[
+        {
+          rel: 'mask-icon',
+          href: '/safari-pinned-tab.svg',
+          color: '#13adc7'
+        },
+        {
+          rel: 'canonical',
+          href: fullUrl
+        }
+      ]}
+    >
+      <script
+        defer
+        data-domain="dvc.org"
+        src="https://plausible.io/js/plausible.outbound-links.js"
+      ></script>
+    </Helmet>
```

Comment:
This is pretty much the entire difference from the theme's original SEO file. The theme itself will follow suit, but there's also the issue of dvc.org being hardcoded that I never noticed before so we'll want to take care of that as well.